lib/mayaUsd/render/vp2RenderDelegate/mayaPrimCommon.cpp

```diff
@@ -319,6 +319,16 @@ TfToken MayaUsdRPrim::_GetMaterialNetworkToken(const TfToken& reprToken) const
     return _displayLayerModes._texturing ? reprToken : TfToken();
 }
 
+HdReprSharedPtr MayaUsdRPrim::_FindRepr(const ReprVector& reprs, const TfToken& reprToken)
+{
+    ReprVector::const_iterator it
+        = std::find_if(reprs.begin(), reprs.end(), [reprToken](ReprVector::const_reference e) {
+              return reprToken == e.first;
```

Comment:
I suppose the vector is not sorted so std::binary_search cannot be used.

Reply:
Those are stored in an RPrim and managed by Hydra. They are not sorted.